docs/hive-integration/import-installed-cluster.md

```diff
@@ -1,84 +1,169 @@
-# Import Installed Cluster
+# Import an existing OCP cluster as an Assisted Day 2 Cluster
 
-It is possible to import an existing installed OpenShift in order to be able to add more workers to it. (Day 2 operation)
+It is possible to take an existing OCP cluster and import this as a Day2 cluster into assisted so that additional workers may be added.
+Additional workers are added by downloading an ISO that has been generated from an InfraEnv linked to the imported cluster, the workers should be booted from this ISO.
```

Comment:
ZTP should be supported as well, so i suggest just t keep the topic on day2 operations.
meaning that user can use a bmh in order to boot the host.